redis/src/cluster_async/mod.rs

```diff
@@ -533,64 +534,116 @@ where
         Ok(())
     }
 
-    async fn execute_on_all_nodes(
+    async fn execute_on_multiple_nodes(
         func: fn(C, Arc<Cmd>) -> RedisFuture<'static, Response>,
         cmd: &Arc<Cmd>,
         only_primaries: bool,
```

Comment:
for a different PR - I think we should pass the routing info to the function and that the function will retrieve its relevant addresses, so it could be used to other groups of nodes besides to PRIMARES/ALL_NODES (e.g. - specific shard, all replicas, group of nodes the user passes..)

Reply:
next PR :)

Reply:
https://github.com/nihohit/redis-rs/pull/7/commits/0a7558da87594d449c658d19233f160703eba525
as requested :)